src/app/modules/react-migration/react-migration.routes.ts

```diff
@@ -7,7 +7,7 @@
 
 ReactMigrationRouter.get(
   '/:formId([a-fA-F0-9]{24})',
-  ReactMigrationController.serveForm,
+  ReactMigrationController.servePublicForm,
```

Comment:
should we rate limit? @timotheeg? or ignore. i think we can rate limit to use our submissions rate limit of 80/second, even though we have cloudflare caching.

Reply:
[cfe98ed](https://github.com/opengovsg/FormSG/pull/5272/commits/cfe98ed858d0072e19c662ed8fa26aa71b102089)

Reply:
Apologies for the delay in commenting here 😱 , this is a GET on the form's html page, I reckon there should be no rate limiting at all 🤔 

The issue is the file system access and we can easily clear that. I'd recommend we load the file from filesystem at bootstrap and keep it in memory. We can then run the replacement on requests which is light.

Just a note on that: loading a filesystem asset into memory is OK to do in an immutable infrastructure. If we were releasing by pushing files on deployment on long-running servers, it wouldn't be the right thing to do, or we'd need additional process of timeout, or filesystem hook to be notified of a change to the file. 

With immutable infra, the file will ever change for the lifetime of the nodejs app. Releases spin a new server, and again the index file don't change for the lifetime of the app, so it's OK to cache in memory.

One more note on that, reading from filesystem is by itself not terrible either. Linux has an excellent filesystem cache, and so for `frontend/index.html`, which has very frequent access, it's very likely the filesystem read call actually gets the content from memory cache rather than actually accessing the disk. Still, since we know the read characteristics, we might as well handle the in-memory cache explicitly in app.